src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/beanvalidation/BeanValidationConstants.java

```diff
@@ -34,7 +34,7 @@ public class BeanValidationConstants {
     public static final String MIN = "jakarta.validation.constraints.Min";
     public static final String MAX = "jakarta.validation.constraints.Max";
     public static final String NEGATIVE_OR_ZERO = "jakarta.validation.constraints.NegativeOrZero";
-    public static final String POSTIVE_OR_ZERO = "jakarta.validation.constraints.PostiveOrZero";
+    public static final String POSTIVE_OR_ZERO = "jakarta.validation.constraints.PositiveOrZero";
```

Comment:
Could you please change the name of the identifier as well? The spelling mistake there may be confusing.

Reply:
I made the changes @turkeylurkey